floodviz/peak_flow_utils.py

```diff
@@ -0,0 +1,102 @@
+import requests
+
+def req_peak_data(site, start_date, end_date, url_pefix):
+    """ 
+    This function first requests water peak flow data in
+    rdb format from NWIS peak water data service.
+
+    ARGS: 
+        site - string site ID for the site to be charted
+        start_date - starting date to chart peak flow data
+        end_date - ending date to chart peak flow data
+        url_pefix - config varbile for nwis peak waterdata service endpoint url
```

Comment:
It's not necessarily a config variable. It's just a string that points to an endpoint that provides the peak streamflow for the site.